packages/strapi-connector-bookshelf/lib/mount-models.js

```diff
@@ -401,7 +391,10 @@ module.exports = async ({ models, target }, ctx, { selfFinalize = false } = {})
             related: function() {
               return this.morphTo(
                 name,
-                ...association.related.map((id, index) => [GLOBALS[id], morphValues[index]])
+                ...association.related.map(morphModel => [
+                  GLOBALS[morphModel.globalId],
```

Comment:
isn't morphModel already the instance of the model ? I think you can return [morphModel, morphModel.collectionName)